<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleScreen</class>
 <widget class="QDialog" name="TitleScreen">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>581</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="HighScores">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>HIGH SCORES</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="exitButton">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="LogoLabel">
      <property name="styleSheet">
       <string notr="true">image:url(:/images/RvrsPacmanLOGO.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="DeveloperLabel">
      <property name="text">
       <string>Developed by: Alexandria (Team 13)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="JoinButton">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
